src/subcommands/logs.ts

```diff
@@ -0,0 +1,160 @@
+// Copyright 2021 Deno Land Inc. All rights reserved. MIT license.
+
+import { wait } from "../../deps.ts";
+import { error } from "../error.ts";
+import { API, APIError } from "../utils/api.ts";
+
+const help = `deployctl logs
+Shows the logs of the given project.
+
+To show the latest logs of a project:
+  deployctl logs --project=helloworld
+
+To show the logs of a particular deployment:
+  deployctl logs --project=helloworld --deployment=1234567890ab
+
+To show the logs of the production deployment:
+  deployctl logs --project=helloworld --prod
```

Comment:
```suggestion
```

I think this is not required as an user can use `--deployment` flag to pass id of a production deployment.